code/modules/clothing/head/helmet.dm

```diff
@@ -45,6 +45,14 @@
 	armor = list(melee = 65, bullet = 30, laser = 50, energy = 10, bomb = 25, bio = 0, rad = 0)
 	flags_inv = HIDEEARS
 
+/obj/item/clothing/head/helmet/hoshelm

```

Comment:
That path is eh.
Use `/obj/item/clothing/head/helmet/hos` for the (new) helmet and `/obj/item/clothing/head/helmet/hos/cap` for the cap and `/obj/item/clothing/head/helmet/hos/dermal` for the dermal armor patch